spec/features/users_new_spec.rb

```diff
@@ -1,9 +1,18 @@
 require 'rails_helper'
 
-describe "User#new page" do
+describe "User#new page", type: :feature do
```

Comment:
This was written before I started BDD and wrote the spinach tests.  I deleted it.